src/@types/index.d.ts

```diff
@@ -2,3 +2,4 @@ declare module 'vue-js-progress';
 interface Window {
   injectedWeb3?: any;
 }
+declare module '@astar-network/astar-ui';
```

Comment:
I think this should be fixed from the `@astar-network/astar-ui` library, not the client. But please correct me if I'm wrong. (it just feels like spaghetti code and too much boilerplating when we wish to use the UI library for a different project)

Reply:
Related to https://github.com/AstarNetwork/astar-apps/pull/291#discussion_r833989566

https://github.com/AstarNetwork/astar-apps/pull/292/files#diff-b880b4c3d2b016a0fe9d248fde4ce65bc863b38df5c2c6200c4844b4e16490f1R5

@sirius651 Could you help advise about this question?

Reply:
yes I know the issue. Will fix on that